RoslynPluginGenerator/AnalyzerPluginGenerator.cs

```diff
@@ -62,10 +68,10 @@ public AnalyzerPluginGenerator(INuGetPackageHandler packageHandler, SonarQube.Pl
             this.logger = logger;
         }
 
-        public bool Generate(NuGetReference analyzeRef, string language, string sqaleFilePath, string outputDirectory)
+        public bool Generate(NuGetReference analyzerRef, string language, string sqaleFilePath, string outputDirectory)
```

Comment:
I considered that but I decided I had made enough boy-scout changes to this method for the time being.